### PR TITLE
Feature/main page frontend

### PR DESCRIPTION
**Pull request by AndrewKozovyi**
UJP-21
UJP-85

### CODE REVIEW

File: frontend/src/pages/mainPage.js [hunk 1]
```diff
@@ -147,8 +147,8 @@ function MainPage() {
                     {sports.map((item, index)=>(
                         <SportBtn
                             key={index}
-                            sport={item.data?.sport_name}
-                            img={item.data?.sport_img}
```
Comment (Iarynovskyi) [inline frontend/src/pages/mainPage.js]: Some durak

File: frontend/src/pages/mainPage.js [hunk 2]
```diff
@@ -1,32 +1,57 @@
...
+    const [games, setGames] = useState([]);
+
+    useEffect(() => {
+        axios.get(`${apiEndpoints.url}${apiEndpoints.news.getRecent}`)
+            .then(res => {
+                const returnedNews = res.data;
+                setNews(returnedNews);
+            })
+            .catch(error => {
+                console.error('There was an error getting news:', error);
```
Comment (Avariq) [inline frontend/src/pages/mainPage.js:24]: Think about implementing some sort of a pop-up (any kind will do) to display those errors to the end user.

Regular customer does not know what console is let alone how to open it

File: frontend/src/pages/mainPage.js [hunk 3]
```diff
@@ -39,28 +64,22 @@ function MainPage({news}) {
...
-                                <h4 className="online">етер</h4>
-
-                                <h1 className="score">0:0</h1>
-
-                                <h4 className="matchLeague">ліга</h4>
-
-                            </div>
-
-                            <i className="fa fa-user-o" aria-hidden="true"></i>
+                        {games.slice(0, 5).map((item, index) => (
```
Comment (Avariq) [inline frontend/src/pages/mainPage.js:69]: Hardcoded values are now welcome unless crucial and required  (I saw a couple places with same values used in the same way).
Consider moving it to some global variable or something like that

File: frontend/src/pages/sportPage.js [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+                    />
+                ))}
+
+            </section>
+
+            <ReactPaginate
+                breakLabel="..."
+                nextLabel="→"
+                onPageChange={handlePageClick}
+                pageRangeDisplayed={3}
```
Comment (Avariq) [inline frontend/src/pages/sportPage.js:98]: I'd also consider having that separated into some variable.

Maybe in future you will want a bigger number here (for example for people with big screens)

File: frontend/src/pages/mainPage.js (hunk 3 above)
Comment (AndrewKozovyi) [reply frontend/src/pages/mainPage.js:69]: That was a test slice by @Romlen2, he's trying to get correct path to the required data

File: frontend/src/pages/mainPage.js (hunk 2 above)
Comment (AndrewKozovyi) [reply frontend/src/pages/mainPage.js]: Done

File: frontend/src/pages/sportPage.js (hunk 4 above)
Comment (AndrewKozovyi) [reply frontend/src/pages/sportPage.js]: I guess, it's done for now

**Review by Avariq — APPROVED** [review]
Approve for now, but pop-up logic should be custom-implemented in future

(Default alerts are sooo 2007...)

**Pull request merged**